vm/src/stdlib/os.rs

```diff
@@ -221,6 +221,31 @@ fn convert_nix_errno(vm: &VirtualMachine, errno: Errno) -> PyClassRef {
     }
 }
 
+fn os_access(path: PyStringRef, mode: PyIntRef, vm: &VirtualMachine) -> PyResult {
+    let mode = mode.as_bigint().to_u8().unwrap();
```

Comment:
You can accept `mode: u8` as a parameter to have it check that the BigInt fits in a u8 for you.

Reply:
I thought only Python types were accepted by functions.

Reply:
[Here's](https://github.com/RustPython/RustPython/pull/1265#discussion_r314192418) a little explanation of how the parameter and return types of native functions work for RustPython. Basically, there's a trait `FromArgs` that lets a type be used as a parameter, and a trait `IntoPyObject` that lets a type be used as a return type.